packages/framework/src/Component/FlashMessage/FlashMessageTrait.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Component\FlashMessage;
+
+/**
+ * @property \Psr\Container\ContainerInterface $container
+ */
+trait FlashMessageTrait
+{
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addSuccessFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addSuccessFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addErrorFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addErrorFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addInfoFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addInfoFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $message
+     */
+    public function addErrorFlash(string $message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_ERROR, $message);
+    }
+
+    /**
+     * @param mixed $message
+     */
+    public function addInfoFlash($message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_INFO, $message);
+    }
+
+    /**
+     * @param mixed $message
+     */
+    public function addSuccessFlash($message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_SUCCESS, $message);
```

Comment:
Is a reason, why `addErrorFlash` is type-hinted as string and  those two are mixed and without explicit type-hint?

Reply:
The reason is that I am a idiot :smile: a622f7219c9df35350be87685179c5201fb4a4aa